PlandoRandomSettings.py

```diff
@@ -67,29 +72,67 @@ def populate_location_exclusions():
     return excluded_locations
 
 
-# Populate starting pool (inventory, songs, equipment)
-def populate_starting_pool(pool):
-    starting_pool = []
-    for item in pool:
-        if random.random() < 0.05:
-            starting_pool.append(pool[item].settingname)
+# Whether to start with Fast Travel
+def start_with_fast_travel():
+    if not hasattr(start_with_fast_travel, "choice"):
```

Comment:
Decided to try this function "attribute", which ensures we get the same answer every time we call start_with_fast_travel().

Reply:
Good call. Better to be safe